### ISSUE TITLE
Time complexity error

### ISSUE THREAD

**Issue by SarangSpin** [DeathStroke19891/CP_Notes#21]
In graph notes, 3rd question on directed acyclic graph ancestors identification the time complexity should be O(V(V+E)) right?

**Comment by DeathStroke19891** [comment]
Yes. Will fix in next graphs PR about LCA and articulation points.
